tests/testthat/test-sce_to_seurat.R

```diff
@@ -34,7 +36,32 @@ test_that("Converting SCE to Seurat objects works as expected", {
 
   # can't directly check that coldata is equal because of added columns in seurat object
   expect_true(all(colnames(coldata_sce) %in% colnames(seurat_object@meta.data)))
-  expect_equal(rowdata_sce, seurat_object[["RNA"]]@var.features)
+  expect_equal(rowdata_sce, seurat_object[["RNA"]]@var.features) # since default "counts" is used, RNA name is expected here
+  expect_equal(metadata(sce), seurat_object@misc)
+
+  # check that altExp data was converted and rowData
+  expect_s4_class(seurat_object[[alt_name]], 'Assay')
+  alt_rowdata_sce <- as.data.frame(rowData(altExp(sce, alt_name)))
+  expect_equal(alt_rowdata_sce, seurat_object[[alt_name]]@var.features)
+
+})
+
+
+test_that("Converting SCE to Seurat objects works as expected for a non-default assay of 'logcounts'", {
+
+  seurat_object <- sce_to_seurat(sce, assay_name = "logcounts")
+
+  # check that column names of Seurat object are derived from SCE object
+  # they won't necessarily be equal if some cells with 0 counts were removed
+  expect_true(all(colnames(seurat_object) %in% colnames(sce)))
```

Comment:
```suggestion
  expect_true(all(colnames(seurat_object) %in% colnames(sce)))
  expect_true("logcounts" %in% seurat_object@assays)
```
I meant doing something like this to make sure that `logcounts` is one of the assays, but this would need to be double checked that calling that gives a list of names.

Reply:
Ah! Sorry, missed that.